apps/mudita-center-e2e/src/specs/overview/e2e-mock-mc-force-update-available.e2e.ts

```diff
@@ -0,0 +1,142 @@
+import { E2EMockClient } from "../../../../../libs/e2e-mock/client/src"
+import modalAppUpdatePage from "../../page-objects/modal-app-update.page"
+import ModalPage from "../../page-objects/modal.page"
+import packageInfo from "../../../../mudita-center/package.json"
+
+describe("Force Update MC - Successful Download", () => {
+  before(async function () {
+    E2EMockClient.connect()
+    //wait for a connection to be established
+    await browser.waitUntil(() => {
+      return E2EMockClient.checkConnection()
+    })
+
+    E2EMockClient.setMockUpdateState({ available: true, version: "999.0.0" })
+    E2EMockClient.mockHttpResponse({
+      url: "v2-app-configuration",
+      method: "GET",
+      status: 200,
+      data: {
+        centerVersion: "999.0.0",
+        productVersions: {
+          MuditaHarmony: "1.0.0",
+          MuditaPure: "1.0.0",
+          APIDevice: "1.0.0",
+        },
+      },
+    })
+  })
+
+  it("Check update modal sections ", async () => {
+    await console.log("PACKAGE INFO VERSION:" + packageInfo.version)
+
+    // Header
+    const modalHeader = await modalAppUpdatePage.modalHeader
+    await expect(modalHeader).toBeDisplayed()
+    await expect(modalHeader).toHaveText("Mudita Center")
+
+    // Available version
+    const paragraphAvailableVersion =
+      await modalAppUpdatePage.paragraphAvailableVersion
+    await expect(paragraphAvailableVersion).toBeDisplayed()
+    await expect(paragraphAvailableVersion).toHaveTextContaining(
+      "Update Mudita Center to"
+    )
+    const textParagraphAvailableVersion =
+      await paragraphAvailableVersion.getText()
+    const availableAppVersion = textParagraphAvailableVersion.split("to ").pop()
+    await console.log("AVAILABLE VERSION:" + availableAppVersion)
```

Comment:
expect(availableVersion) to be same as E2EMockClient.setMockUpdateState.version ?